packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -0,0 +1,145 @@
+import { ICheckStyleProps, ICheckStyles } from './Check.types';
+import {
+  concatStyleSets,
```

Comment:
Some of these imports aren't needed. Not sure why no-unused-imports isn't picking this up.

Reply:
Hmm, me either. I'll take the unused ones out.